src/utils.ts

```diff
@@ -76,3 +76,32 @@ export function generateIssueComment(claims: BotClaimData[]): string {
 
   return comment;
 }
+
+// must be a valid mention of any tags (`@gitpoap-bot` or any username).
+// need whitespace or punctuation surrounding, or at least at the end if it's the first thing in the text
+export const hasGitPoapBotTagged = (comment: string): boolean => {
+  // found gitpoap-bot with spaces and quotes
+  if (
+    comment.includes(' @gitpoap-bot ') ||
+    comment.includes('"@gitpoap-bot"') ||
+    comment.includes("'@gitpoap-bot'")
+  )
+    return true;
+  // found gitpoap-bot at the beginning with space
+  if (comment.includes('@gitpoap-bot ') && comment.indexOf('@gitpoap-bot') === 0) return true;
```

Comment:
Seems like this would miss things like:
* `@gitpoap-bot: @tyler415git`
* `@gitpoap-bot, please award @tyler415git`